src/AniLink/Fade.js

```diff
@@ -19,9 +19,8 @@ const fade = ({ exit: { length }, node, direction }) => {
 			  }
 			: {}
 
-	return new TimelineMax()
-		.set(node, holdPosition)
-		.fromTo(node, duration, { opacity: !opacity }, { opacity: opacity })
+	return gsap.set(node, holdPosition)
```

Comment:
This should be

```suggestion
	return gsap.timeline()
		.set(node, holdPosition)
```

But the fade animation is still broken after fixing this 🤔 looks like in Fade.js the TransitionLink component needs to have the zIndexes set for some reason.

```js
<TransitionLink
  exit={{
    length: length,
    zIndex: 1,
    trigger: ({ exit, node }) => fade({ exit, node, direction: "out" })
  }}
  entry={{
    length: 0,
    zIndex: 1,
    trigger: ({ exit, node }) => fade({ exit, node, direction: "in" })
  }}
  {...props}
>
  {props.children}
</TransitionLink>
```
Then it works

Reply:
It seems to work fine with just the exiting node places high in the z plane so I've added it just for that.